engines/dfc_provider/script/update-dfc-vocabularies

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+dst="`dirname $0`/../vendor"
+mkdir -p "$dst"
+
+curl 'http://static.datafoodconsortium.org/data/facets.json' > "$dst/facets.json"
+curl 'http://static.datafoodconsortium.org/data/measures.json' > "$dst/measures.json"
+curl 'http://static.datafoodconsortium.org/data/productTypes.json' > "$dst/productTypes.json"
```

Comment:
Shouldn't we run this periodically? On startup?

Reply:
Or maybe it should be tied to a version?

Reply:
The vocabularies are not versioned, unfortunately. But we should run this without review and test because we may be referring product categories as Ruby classes for example. So these vocabularies change very rarely but can change at any time and could contain breaking changes while not being versioned. The update script is only for a developer's convenience.